tests/functional/modules/test_zos_find_func.py

```diff
@@ -15,6 +15,7 @@
 __metaclass__ = type
 
 from ibm_zos_core.tests.helpers.volumes import Volume_Handler
+import pytest
```

Comment:
Is this import needed?

Reply:
Yes, since I added a parametrize section in one test. Otherwise will fail like this 

```
_____________________________________________ ERROR collecting functional/modules/test_zos_find_func.py _____________________________________________
tests/functional/modules/test_zos_find_func.py:222: in <module>
    @pytest.mark.parametrize("ds_type", DATASET_TYPES)
E   NameError: name 'pytest' is not defined
```